simplematrixbotlib/api.py

```diff
@@ -307,3 +307,48 @@ async def send_image_message(self, room_id, image_filepath):
             await self._send_room(room_id=room_id, content=content)
         except:
             print(f"Failed to send image file {image_filepath}")
+
+    async def send_video_message(self, room_id, video_filepath):
+        """
+        Send a video message in a Matrix room.
+
+        Parameters
+        ----------
+        room_id : str
+            The room id of the destination of the message.
+
+        video_filepath : str
+            The path to the video on your machine.
+        """
+
+        mime_type = mimetypes.guess_type(video_filepath)[0]
+
+        file_stat = await aiofiles.os.stat(video_filepath)
+        async with aiofiles.open(video_filepath, "r+b") as file:
+            resp, maybe_keys = await self.async_client.upload(
+                file,
+                content_type=mime_type,
+                filename=os.path.basename(video_filepath),
+                filesize=file_stat.st_size)
+
+        if isinstance(resp, UploadResponse):
+            pass # Successful upload
+        else:
+            print(f"Failed Upload Response: {resp}")
+
+        content = {
+            "body": os.path.basename(video_filepath),
+            "info": {
+                "size": file_stat.st_size,
+                "mimetype": mime_type,
+                "thumbnail_info": None
```

Comment:
https://spec.matrix.org/v1.3/client-server-api/#mvideo

I think clients such as Nheko don't play videos that don't have e.g. a duration key.

We should be able to (generally) use [pymediainfo](https://pypi.org/project/pymediainfo/) which is also used by [nemo media columns](https://github.com/linuxmint/nemo-extensions/tree/master/nemo-media-columns), however that of course leads to dependency trouble similar to that with libolm as it requires the (non-python) library to be installed.

Reply:
Most of the solutions that I have found involve require ffmpeg or something similar. https://zulko.github.io/moviepy/install.html claims to install ffmpeg automatically, but that would need to be tested.

Reply:
Thoughts on adding an "additional requirements" section to the "getting started" section of the docs with a link to that from the docs for send_video_message?

We could then add pymediainfo and https://mediaarea.net/en/MediaInfo as required dependencies for sending video messages.